assets/js/script.js

```diff
@@ -19,6 +22,32 @@ function checkForm(form) {
   return false;
 }
 
+/* localStorage */
+let formData = { name: '', email: '', message: '' };
+
+const onChangeHandler = (event) => {
+  switch (event.name) {
+    case 'name':
+      formData = { ...formData, name: event.value };
+      break;
+    case 'email':
+      formData = { ...formData, email: event.value };
+      break;
+    case 'message':
+      formData = { ...formData, message: event.value };
+      break;
+    default:
+      break;
+  }
+  localStorage.setItem('data', JSON.stringify(formData));
+};
+const reloadBrowser = JSON.parse(localStorage.getItem('data'));
+if (reloadBrowser) {
+  formName.value = reloadBrowser.name;
+  email.value = reloadBrowser.email;
+  message.value = reloadBrowser.message;
```

Comment:
**Changes Required**
- [ ] As mentioned before, you have a bug here that clears the `localStorage` every time you submit the form. Let us try and fix this to have to achieve correctness.